build.gradle

```diff
@@ -28,35 +53,14 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.0.0-SNAPSHOT'
-    compile group: 'org.opentest4j', name: 'opentest4j', version: '1.0.0-M1'
-
-    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.0.0-SNAPSHOT'
-    testCompile group: 'org.junit.platform', name: 'junit-platform-launcher', version: '1.0.0-SNAPSHOT'
+    api group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.0.0'
```

Comment:
We've never discussed versions so far, but my goal was to continuously build against the latest snapshot and only pick a concrete version when creating a release branch. For now, can you please use the latest snapshots (5.1... I assume)?

Reply:
Might not be a bad idea to have the CI run builds against both 'latest' and static versions. I changed them to the release because it could be error prone if accidentally published and because when I pulled this project up on the train it tried to refresh those dependencies :neutral_face: 

I'll switch it back to the snapshot for this PR

Reply:
Just created #64 to discuss this.

> Might not be a bad idea to have the CI run builds against both 'latest' and static versions.

Yes, that's a good idea.

> it could be error prone if accidentally published

True, would be good to build something into the Gradle script that checks this.

> when I pulled this project up on the train it tried to refresh those dependencies

Didn't think about that, you're right it makes the project depend on being online - I actually like working in the train, so that's a serious downside.